molo/yourtips/templates/yourtips/your_tips_entry.html

```diff
@@ -1,17 +1,18 @@
 {% extends "base.html" %}
 {% load wagtailcore_tags wagtailimages_tags tip_tags static core_tags i18n %}
 {% block content %}
+{% get_your_tip as your_tip %}
 <div class="yourtips yourtips--page">
   <h1 class="heading heading--yourtips-page">
     {% trans 'Submit' %} {{tip_page.title|slice:":-1"}}
   </h1>
   <h3 class="yourtips__description">
-    {% if homepage_action_copy %}
-      {% trans homepage_action_copy %}
+    {% if your_tip.homepage_action_copy %}
+      {% trans your_tip.homepage_action_copy %}
```

Comment:
@cblignaut we don't need to wrap `your_tip.homepage_action_copy` variable in a trans block, trans block won't work on the variables as you need the text when generating a PO file. In this case admin will need to add the translated text in wagtail.